draft-ietf-quic-multipath.md

```diff
@@ -198,7 +207,7 @@ A multipath QUIC connection starts with a QUIC handshake as a regular QUIC conne
 See further {{nego}}.
 The peers use the enable_multipath transport parameter during the handshake to
 negotiate the utilization of the multipath capabilities.
-The active_connection_id_limit transport parameter limits the maximum number of active paths
+The max_concurrent_paths transport parameter limits the maximum number of active paths
```

Comment:
This section was meant to explain the meaning of active_connection_id_limit when this extension is used. we still need that. I think with this PR it would apply per path...?

Reply:
Yes, `active_connection_id_limit` would apply per path in my proposal (#214).

Reply:
Solved that.